querybuilder/evaluator_test.go

```
@@ -29,7 +29,7 @@ func TestMatch(t *testing.T) {
 
 	for _, input := range inputs {
 		t.Run(input.title, func(t *testing.T) {
-			if got := qb.Match(parseJson(input.dataset)); got != input.want {
+			if got, _ := qb.Match(parseJson(input.dataset)); got != input.want {
```

no need to suppress errors